.github/workflows/reacton-classroom-ci-cd.yml

```
@@ -55,40 +55,52 @@ jobs:
-            rm -f .env
+            
+            echo "🛑 Stopping and removing existing container..."
+            docker stop reacton-classroom-back || true
+            docker rm reacton-classroom-back || true
+            
+            echo "🔄 Logging in to Docker Hub..."
+            docker login -u ${{ secrets.DOCKER_USERNAME }} -p ${{ secrets.DOCKER_PASSWORD }}
+
+            echo "🔄 Pulling latest Docker image..."
+            docker pull ${{ secrets.DOCKER_USERNAME }}/reacton-classroom-back:latest
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)